cmd/draft/delete.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+
+	"github.com/Azure/draft/pkg/draft/draftpath"
+	"github.com/Azure/draft/pkg/draft/local"
+)
+
+const deleteDesc = `This command deletes an application from your Kubernetes environment.`
+
+type deleteCmd struct {
+	appName string
+	out     io.Writer
+	home    draftpath.Home
+}
+
+func newDeleteCmd(out io.Writer) *cobra.Command {
+	dc := &deleteCmd{
+		out: out,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "delete [app]",
+		Short: "delete an application",
+		Long:  deleteDesc,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				dc.appName = args[0]
+			}
+			return dc.run()
+		},
+	}
+
+	dc.home = draftpath.Home(homePath()) // TODO: do I actually need this?
```

Comment:
I don't think this needs to be in here given that we never need to actually pull anything from ~/.draft in `draft delete`. I don't see any other references to home other than the unit tests. :)

Reply:
you really have a good eye for catching my TODOs lol thank you!